taf/updater/updater.py

```diff
@@ -977,6 +914,66 @@ def _set_target_old_head_and_validate(
     return old_head
 
 
+def _run_tuf_updater(git_updater):
+    def _init_updater():
+        try:
+            return Updater(
+                git_updater.metadata_dir,
+                "metadata/",
+                git_updater.targets_dir,
+                "targets/",
+                fetcher=git_updater,
+            )
+        except Exception as e:
```

Comment:
Don't think it's necessary unless we also want to log the error

Reply:
Added log